recipes/raxml-ng/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "raxml-ng" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/amkozlov/{{ name|lower }}.git
+  git_rev: {{ version }}
```

Comment:
Do you need to use git rather than https://github.com/amkozlov/raxml-ng/archive/0.9.0.tar.gz ?

Reply:
There are two submodules hidden in the libs/ subdirectory that this deals with nicely